extensions/promise/promise.js

```diff
@@ -621,4 +621,26 @@ Using '.thenAll' makes this a bit smoother:\
         
         return promisedType;
     }
+
+    Promise._help_any = {
+        description: "Similar to Promise.all, except it resolves when a single promise provided has succeeded.",

```

Comment:
I don't think it similar to Promise.all at all. :)

I think it would be better to say something along the lines of: "Creates a single promise that resolves when any one of the promises in the given iterable resolve."